File: goodReads.client/src/models/Account.js
export class Account {
  constructor(data) {
    this.id = data.id || data._id
    this.email = data.email
    this.name = data.name.split("@")[0]
    this.picture = data.picture
    this.bio = data.bio
    // TODO add additional properties if needed
  }
}

File: goodReads/Models/Account.cs
namespace goodReads.Models;

public class Profile
{
  public string Id { get; set; }
  public string Name { get; set; }
  public string Picture { get; set; }
  public string Bio {get; set;}

}

public class Account : Profile
{
  public string Email { get; set; }
  public bool?  Theme { get; set; } = false;

}